edward/inferences/inference.py

```diff
@@ -210,6 +214,18 @@ def initialize(self, n_iter=1000, n_print=None, scale=None, logdir=None,
 
     self.scale = scale
 
+    if auto_transform:
+      latent_vars = self.latent_vars.copy()
+      self.latent_vars = {}
+      for z, qz in six.iteritems(latent_vars):
+        try:
+          if z.support != qz.support:
```

Comment:
If the supports are matched, nothing happens.
If the supports are not matched, individual z,qz pairs get transformed.

Does it make sense to log/print/report these somehow? I can see this getting challenging to debug going forward?

Reply:
Yep, I agree. It's sort of open how to report these automations so we haven't gotten around to it. Other examples are the default optimizer and the default posterior approximations in ed.MonteCarlo and ed.MAP.

Maybe we can establish such a standard in a new PR. I added an issue #764.